src/argowrapper/engine/argo_engine.py

```diff
@@ -538,7 +538,7 @@ def get_workflow_logs(self, workflow_name: str, uid: str) -> List[Dict]:
             uid (str): uid of an archived workflow to get status of
 
         Returns:
-            Dict[str, any]: returns a list of dictionaries of errors of Retry nodes
+            Dict[str, Any]: returns a list of dictionaries of errors of Retry nodes
```

Comment:
issue not introduced here, but visible now: the docmented return does not match the return type in the method signature: `-> List[Dict]`

Reply:
Fixed this one. 👍